components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -79,6 +108,14 @@ export class TeamDBImpl implements TeamDB {
             throw new Error('Please choose a team name containing only letters, numbers, -, _, \', or spaces.');
         }
         const slug = name.toLocaleLowerCase().replace(/[ ']/g, '-');
+        if (FORBIDDEN_SLUGS.indexOf(slug) !== -1) {
+            throw new Error('Creating a team with this name is not allowed');
+        }
+        const userRepo = await this.getUserRepo();
+        const existingUsers = await userRepo.query('SELECT COUNT(id) AS count FROM d_b_user WHERE fullName LIKE ? OR name LIKE ?', [ name, slug ]);
+        if (existingUsers[0].count > 0) {
+            throw new Error('A team cannot have the same name as an existing user');
```

Comment:
**question**: Do we want to surface this kind of information? It seems this is privacy sensitive information, right? Maybe using a more generic message could be better? What do you think? 💭 

```suggestion
            throw new Error('Team path is already in use');
```

Reply:
Hmm, not sure about the privacy aspect. If someone wants to create a team called "Jan Keromnes", and it fails, I'd be okay with the error message saying that a user is already called that name. 😋

Reply:
Also, user names come directly from GitHub or GitLab, where they're already public information anyway.

Reply:
I still think this something we could consider changing in the future but it should be fine shipping this as is for now! 🙈

Reply:
What I'd love in the future is public Gitpod profiles, e.g. showcasing some public Gitpod stuff you'd like to share with the world. 🙂 Could give Gitpod more of a "social" vibe.

Reply:
This is one of the reasons I'd vote for making this more privacy-aware but should be fine to skip this for now.